source/Ship.cpp

```diff
@@ -2504,7 +2505,8 @@ double Ship::IdleHeat() const
 	// heat = heat * diss + heatGen - cool - activeCool * heat / (100 * mass)
 	// heat = heat * (diss - activeCool / (100 * mass)) + (heatGen - cool)
 	// heat * (1 - diss + activeCool / (100 * mass)) = (heatGen - cool)
-	double production = max(0., attributes.Get("heat generation") - cooling);
+	double production = max(0., attributes.Get("heat generation") 
+		+ attributes.Get("solar heat") - cooling);
```

Comment:
Looks like fuel heat isn't added here despite the fact that it is factored into the heat table in ShipInfoDisplay. Add it here as well, or remove solar heat?

Reply:
I don't know that we want to modify IdleHeat like this, since this is ignoring all components that otherwise influence the per frame added heat (system solar power and range from center). Probably it is OK to add the maximal solar heat value, rather than actually compute the scaled power factor.

As far as adding "fuel heat", that should require a conditional check since fuel heat is only added to the ship's heat when the ship also has both "fuel consumption" and enough fuel to be consumed that frame. The computation is already stateful--a ship can report a different IdleHeat at different game frames due to changes in mass--so updating it with dependencies on fuel, fuel consumption, and the energy needed for active cooling is fine. The function is only used when a ship is instantiated in-flight, lands at a spaceport, or when performing an overheating flightcheck, so this calculation can be a little more correct without negatively affecting performance.

Comparing the values of IdleHeat before and after adding in those values, for ships with active cooling, could be interesting from an academic standpoint, along with comparing the value of `heat - IdleHeat()` for one of these ships that launches (and does nothing).

Reply:
I'll just remove this change.